Resources/Prototypes/DeltaV/Entities/Objects/Devices/pda.yml

```diff
@@ -81,3 +81,36 @@
   - type: Icon
     sprite: DeltaV/Objects/Devices/pda.rsi
     state: pda-mailcarrier
+
+- type: entity
+  parent: BasePDA
+  id: AdminAssistantPDA
+  name: administrative assistant PDA
+  description: Theres pen scribbles all over the edges, and a few sticky notes stuck on it.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Devices/pda.rsi
+    layers:
+    - map: [ "enum.PdaVisualLayers.Base" ]
+    - state: "light_overlay"
+      map: [ "enum.PdaVisualLayers.Flashlight" ]
+      shader: "unshaded"
+      visible: false
+    - state: "id_overlay"
+      map: [ "enum.PdaVisualLayers.IdLight" ]
+      shader: "unshaded"
+      visible: false
+  - type: Pda
+    id: AdminAssistantIDCard
+    state: pda-admin-assistant
+    penSlot:
+      startingItem: PenCap
```

Comment:
Shouldn't have Captain's pen if they aren't Captain. I'd suggest the default pen or the luxury pen if you're inclined

Reply:
Gave them a luxury pen instead